### DISCUSSION TITLE
Received 65 cert requests for an unknown ca

### DISCUSSION THREAD

**Discussion by krcnpeter** [strongswan/strongswan#2474] [Q&A]
Hi, I've been having this problem for a week now and I can't get a handle on it.

I have correctly created via PKI tool CA certificate, server certificate, client certificate but when I try to connect in the log there is this error:

```
Sep 29 10:01:21 KRCN-NetCloud01 charon[31585]: 05[IKE] received cert request for "C=SK, O=KRCN.Services, CN=NetCloud01@KRCN.Services IKE-Root CA"
Sep 29 10:01:21 KRCN-NetCloud01 charon[31585]: 05[IKE] received 65 cert requests for an unknown ca
Sep 29 10:01:21 KRCN-NetCloud01 charon[31585]: 05[IKE] received end entity cert "C=SK, O=KRCN.Services, CN=krcn.peter"
```

I'm still using ipsec.conf and ipsec.secrets
The certificates are RSA - the private keys are 4096-bit

I used this instruction to create it https://docs.strongswan.org/docs/5.9/pki/pkiQuickstart.html

ipsec.conf: 
```

conn ikev2-vpn
    auto=add
    compress=no
    type=tunnel
    keyexchange=ikev2
    fragmentation=yes
    forceencaps=yes
    dpdaction=clear
    dpddelay=300s
    rekey=no
    mobike=yes
    left=01.netcloud.krcn.services
    leftid=@01.netcloud.krcn.services
    leftcert=server-cert.pem
    leftsendcert=always
    leftsubnet=0.0.0.0/0
    right=%any
    rightid=%any
    rightsourceip=172.28.128.240/28
    rightdns=5.75.254.73,49.13.122.149
    rightsendcert=never
    ike=aes256-sha256-modp2048!
    esp=aes256-sha256!
```

ipsec.secrets
`
: RSA "server-key.pem"`

**Comment by tobiasbrunner** [comment]
That's **not** an error message. It just tells you that the peer sent you a list of unknown CA certificate public key hashes, besides the one for `C=SK, O=KRCN.Services, CN=NetCloud01@KRCN.Services IKE-Root CA` that the daemon actually recognized. Depending on how the client is configured and what CA certificates it has loaded, the list can be quite long, which in turn can also make the IKE_SA_INIT request quite large as each of these SHA-1 hashes requires 20 bytes.
